src/benchmark_mediation.py

```diff
@@ -956,25 +959,48 @@ def med_dml(
     ----------
     x : array-like, shape (n_samples, n_features_covariates)
         Covariates value for each unit, multidimensional or continuous.
+
     t : array-like, shape (n_samples)
         Treatment value for each unit.
+
     m : array-like, shape (n_samples, n_features_mediator)
         Mediator value for each unit, multidimensional or continuous.
+
     y : array-like, shape (n_samples)
         Outcome value for each unit.
+
+    forest : boolean, default=False
+        Whether to use a random forest model to estimate the propensity
+        scores instead of logistic regression, and outcome model instead
+        of linear regression.
+
     crossfit : int, default=0
         Number of folds for cross-fitting.
+
     trim : float, default=0.05
         Trimming treshold for discarding observations with extreme probability.
+
     normalized : boolean, default=True
         Normalizes the inverse probability-based weights.
+
     regularization : boolean, default=True
         Whether to use regularized models (logistic or linear regression).
         If True, cross-validation is used to chose among 8 potential
         log-spaced values between 1e-5 and 1e5.
+
     random_state : int, default=None
         LogisticRegression random state instance.
 
+    calibration : boolean, default=True
+        Whether to add a calibration step so that the classifier used to estimate
+        the treatment propensity score and P(T|M,X).
+        Calibration ensures the output of the [predict_proba](https://scikit-learn.org/stable/glossary.html#term-predict_proba)
+        method can be directly interpreted as a confidence level.
+
+    calib_method : str, default="sigmoid"
```

Comment:
The two parameters are redundant. Settinf Calib_method to False generally means 'no calibration'